packages/devtools_app/lib/src/screens/memory/panes/profile/model.dart

```diff
@@ -10,33 +10,43 @@ import '../../../../shared/table/table_data.dart';
 import '../../../vm_developer/vm_service_private_extensions.dart';
 import '../../shared/heap/class_filter.dart';
 
+class _ProfileJson {
```

Comment:
Use an enum instead

Reply:
If I use enum, I will have to add `.name` each time i need string value that will crowd serialization code.